package.json

```diff
@@ -72,19 +82,23 @@
     "vm-browserify": "^1.0.0",
     "xtend": "^4.0.0"
   },
+  "//devDependencies": {
+    "//": "Comments about why some devDependencies are at specific versions that may issue warnings.",
+    "rimraf@2.6.3": "Dependency of temp. No update for temp available."
+  },
   "devDependencies": {
     "backbone": "~0.9.2",
     "browser-unpack": "^1.1.1",
-    "coffee-script": "~1.10.0",
-    "coffeeify": "~1.1.0",
+    "coffeescript": "^2.7.0",
+    "coffeeify": "^3.0.1",
     "has-object-spread": "^1.0.0",
     "has-template-literals": "^1.0.0",
     "isstream": "^0.1.2",
     "make-generator-function": "^1.1.0",
     "semver": "^5.5.0",
     "seq": "0.3.5",
-    "tap": "^10.7.2",
-    "temp": "=0.8.3",
+    "tap": "^21.0.1",
+    "temp": "^0.9.4",
```

Comment:
This is pinned on purpose, because v0.8.3 supports node 0.8 but v0.8.4 requires node 6.